presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java

```diff
@@ -90,6 +90,11 @@ public WindowNode(
         return ImmutableList.copyOf(concat(source.getOutputSymbols(), windowFunctions.keySet()));
     }
 
+    public Set<Symbol> getCreatedSymbols()
```

Comment:
Please see https://github.com/prestodb/presto/pull/7298#discussion_r100124840.
@arhimondr agrees with the proposed name.